app/views/admin/products/show.html.erb

```diff
@@ -1,18 +1,2 @@
-<h2 class='text-primary'>PRODUCT ID: <%= @product.id %></h2><br/>
-<div class='container'>
-  <strong>Name of Product:</strong>
-  <%= @product.title %><br/>
-  <hr/>
-  <strong>Price of Product:</strong>
-  <%= @product.price %><br/>
-  <hr/>
-  <strong>Description of Product:</strong>
-  <%= @product.description %><br/>
-  <hr/>
-  <strong>Coupons with this product</strong><br/>
-  <% @product.coupons.each do |coupons| %>
-    <%= coupons.id %>
-    <%= coupons.name %><br/>
-  <% end %>
-  <hr/><br/><%= link_to 'Back', admin_products_path, class: 'btn btn-primary' %>
-</div>
+<%= render 'show' %>
```

Comment:
Why we have created this partial? Content was fine here. As partial takes times to render itself.